staging/2021/benchmarks/json/wi-benchmarks.json

```diff
@@ -0,0 +1,135 @@
+[
+  {
+    "measureId": "110",
+    "benchmarkYear": 2021,
```

Comment:
@lavioli-work I believe we need to make all of the `benchMark` years `2019` across the board, could you make this change?